docs/operator-manual/upgrading/1.7-1.8.md

```diff
@@ -27,12 +27,14 @@ data:
     argoproj.io/Application:
       health.lua: |
         hs = {}
-        hs.status = "Healthy"
+        hs.status = "Progressing"
```

Comment:
🤦 how could I miss it

Reply:
it still not working with me even after changing the status to Progressing, is there any other configurations I need to update?